File: apiEZPZ/src/main/java/com/example/apiezpz/rating/domain/Airport.java
package com.example.apiezpz.rating.domain;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Getter
@Setter
@NoArgsConstructor
public class Airport {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String name;
    private String code;
}

File: apiEZPZ/src/main/kotlin/com/example/apiezpz/rating/domain/Airport.kt
package com.example.apiezpz.rating.domain

import jakarta.persistence.*

@Entity
@Table(name = "airport")
data class Airport(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long? = null,

    @Column(unique = true, nullable = false)
    var name: String = "",
    
    @Column(nullable = false)
    var code: String = ""
) 